xgux.h

```diff
@@ -91,6 +91,20 @@ void xgux_set_clear_rect(unsigned int x, unsigned int y,
     pb_end(p);
 }
 
+#define NV097_SET_CONTROL0_TEXTUREPERSPECTIVE 0x100000
```

Comment:
Maybe this would be a better fit in [nv2a_regs.h#L808](https://github.com/dracc/xgu/blob/2b1e4e4cc125ed7a8ba9f0eb026bc163f54825ba/nv2a_regs.h#L808) ?